### DISCUSSION TITLE
Adding users to a sub-org

### DISCUSSION THREAD

**Discussion by kameshbhr** [Sunbird-Lern/Community#56] [Query]
In this [page](https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service) about User and Orgs, it says "Each user should be mapped to the custodian or other tenant orgs."

Does it mean a user cannot be mapped to a sub-org?

**Comment by BharathwajShankar** [comment]
HI @kameshbhr can you bit elaborate what do you mean by sub-org?

**Reply by kameshbhr** [reply]
A subordinate level organization inside a tenant organization. Please refer to: https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service/how-to-guide/create-organization#taskflow

**Reply by reshmi-nair** [reply]
@kameshbhr A user has to belong to a tenant org. If we do not specify a tenant org while creating a user, by default the user is mapped to custodian org. Currently in sunbird user may or may not choose a sub-org. It is not mandatory.
